src/commcare_cloud/commands/deploy.py

```diff
@@ -112,7 +113,7 @@ def run_ansible_playbook_command():
             )
         rc = run_ansible_playbook_command()
         if rc != 0:
-            return rc
+            return
```

Comment:
My recollection is that these `CommandBase` subclasses are supposed to return the response code that we want the process to exit with. In this case the choice was made that if the ansible deploy exits with a non-zero status, then we want the cchq deploy command to exit with that same status. I'm not sure what behavior returning `None` here results in. Can you fill me in on your thinking behind this change, what prompted it, etc.?

Reply:
I think this function is only called in one place, and the return code is ignored

https://github.com/dimagi/commcare-cloud/blob/b20e3cec40b01d0c86f923d893d80b453debf0b2/src/commcare_cloud/commands/deploy.py#L91

Is there some magic happening somewhere that is calling this in a place I didn't notice?

Reply:
From @dannyroberts response
> I would have fixed it by having the calling function you noted return the value it returns

That call is in an `if` block, so the return value of the `run()` method would be inconsistent, sometimes returning a value and sometimes not depending on the type of deploy.